enigma/src/main/java/org/quiltmc/enigma/api/translation/mapping/tree/MergedEntryMappingTree.java

```diff
@@ -42,10 +43,15 @@ public EntryMapping remove(Entry<?> entry) {
 	@Override
 	public EntryMapping get(Entry<?> entry) {
 		EntryMapping main = this.mainTree.get(entry);
+
 		if (main == null || (main.equals(EntryMapping.OBFUSCATED) && this.secondaryTree.contains(entry))) {
 			return this.secondaryTree.get(entry);
 		}
 
+		if (main.tokenType().equals(TokenType.OBFUSCATED) && this.secondaryTree.contains(entry)) {
```

Comment:
@ix0rai what is your opinion on this? it allows for javadocs to be on empty mappings, but get a name from a proposer

Reply:
just making sure you know i'm not ignoring you, i'm pretty busy rn and i want to look into javadoc support on obf mappings a bit deeper